src/utils/data/MeetingHelper.ts

```diff
@@ -41,37 +40,11 @@ export class MeetingHelper extends CloudBase {
   }
 
   createEvent(subj: string = '', start: Moment = moment(), duration: Duration = moment.duration(1, 'hour'), participants: Participant[] = []): Promise<any> {
-
-    const attendees = participants.map(participant => (
-      {
-        type: 'required',
-        emailAddress: {
-          name: participant.name,
-          address: participant.email
-        }
-      }
-    ));
-
-    const eventData = {
-      originalStartTimeZone: 'UTC',
-      originalEndTimeZone: 'UTC',
-      subject: subj,
-      sensitivity: 'normal',
-      isAllDay: false,
-      responseRequested: true,
-      showAs: 'busy',
-      type: 'singleInstance',
-      body: {contentType: 'text', content: 'hello from helper'},
-      start: {dateTime: moment.utc(start), timeZone: 'UTC'},
-      end: {dateTime: moment.utc(start.clone().add(duration)), timeZone: 'UTC'},
-      location: {displayName: 'helper', address: {}},
-      attendees,
-    };
-    console.log(JSON.stringify(eventData));
-    return this.client.api(`/users/${this.email}/calendar/events`).post(eventData) as Promise<any>;
+    assert(participants.length === 1);
+    return this.meetingsSvc.createEvent(subj, start, duration, {name: 'no used', email: this.email}, participants[0]);
```

Comment:
The owner name is actually used. We should replaced "no used" with a real-sounding name.

![screen shot 2017-04-07 at 8 47 08 pm](https://cloud.githubusercontent.com/assets/12530451/24824052/6a6c5da8-1bd3-11e7-8c34-2f324df45aa4.png)